frontend/src/api/k8s/templates.ts

```diff
@@ -1,6 +1,34 @@
-import { k8sListResource } from '@openshift/dynamic-plugin-sdk-utils';
-import { TemplateKind } from '~/k8sTypes';
-import { TemplateModel } from '~/api/models';
+import YAML from 'yaml';
+import { k8sCreateResource, k8sListResource } from '@openshift/dynamic-plugin-sdk-utils';
+import { ServingRuntimeKind, TemplateKind } from '~/k8sTypes';
+import { ServingRuntimeModel, TemplateModel } from '~/api/models';
+import { applyK8sAPIOptions } from '~/api/apiMergeUtils';
+
+export const assembleTemplate = (body: string, namespace: string): TemplateKind => {
+  const servingRuntime = YAML.parse(body);
+
+  const displayName = servingRuntime.metadata?.annotations?.['display-name'];
```

Comment:
Follow our standard display name annotation please.
```
resource.metadata.annotations?.['openshift.io/display-name'] || resource.metadata.name;
```

Consider using the `getDisplayNameFromK8sResource`.

Please reach out to me before implementing "new terminology" -- we should aim to stay aligned on all fronts -- don't let outside forces influence the "how" we do something.

Reply:
Agh, I'm sorry, thanks, that was a placeholder that I missed to review before raising the PR, changing it right now.